rc/kak-dap.kak

```diff
@@ -292,3 +292,14 @@ define-command -hidden dap-stack-trace -params 3 %{
 define-command -hidden dap-evaluate-response -params 2.. %{
     try %{ eval -client %opt{jumpclient} %{ info -title "Result" " %arg{1}:%arg{2} "}}
 }
+
+declare-user-mode dap
+map global dap s -docstring 'start' ': dap-start<ret>'
+map global dap b -docstring 'toggle breakpoints' ': dap-toggle-breakpoint<ret>'
+map global dap c -docstring 'continue' ': dap-continue<ret>'
+map global dap n -docstring 'next' ': dap-next<ret>'
+map global dap o -docstring 'step out' ': dap-step-out<ret>'
+map global dap i -docstring 'step in' ': dap-step-in<ret>'
+map global dap e -docstring 'eval' ': dap-evaluate '
```

Comment:
I'd get rid of this. The point of `dap-evaluate` is to send an expression to the debug adapter, so it can evaluate said expression and show the output to the user. This can't be done with a fixed keybinding, as the expression evaluated (in this case, none) will always be the same.

Reply:
The binding doesn't actually execute the command - notice the lack of <ret>. It simply populates your command bar with the command name, then lets you type the rest.

In that vein, I don't think it should have the space between the colon and command, because you probably want it to stay in your command history.

Reply:
Ah - I didn't notice that. Never mind then.